app/controllers/custom_fields_controller.rb

```diff
@@ -1,7 +1,7 @@
 class CustomFieldsController < ApplicationController
   include InputSanitizeHelper
 
-  before_action :load_vars, only: [:update, :destroy, :destroy_html]
+  before_action :load_vars, except: :create
```

Comment:
reminder: make a double check if this class get decorated by some addon because if it is the variables will be loaded in this methods to and maybe we will face some strange behavior in the app like with team_user